docs/contributions/key-concepts-and-configurations.md

```diff
@@ -127,6 +127,20 @@ edit existing acceptable credit cards or define a new option through
     then you will need to configure accepted credit cards/payment methods on 
     that site.
 
+## Status
+
+All Contributions have one Status, the most common are:
+
+**Pending (Pay Later)** indicates a contribution that has been entered but payment is not yet received.  This is usually by paper or electronic check.
```

Comment:
@Stoob not sure but maybe also include "Pay Later event registrations" as another reason of this status?

Reply:
Perhaps but this is under the Contributions section, and contributions could be Event Fee, Donations, or any number of other financial transactions.  Do we need to mention events?